packages/vue/src/view/article/ArticleLink.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <a v-bind:href="articleUrl" v-bind:class="className">
+    <slot />
+  </a>
+</template>
+
+<script>
+  export default {
+    props: ['article', 'className'],
```

Comment:
You don't need to use props classname, Vue already put in your component for example

`<component class="name" />`